mne/viz/evoked.py

```diff
@@ -897,12 +897,34 @@ def plot_evoked_white(evoked, noise_cov, rank=None, show=True):
 
 
 def _match_proj_type(proj, ch_names):
-    """see if proj should be counted"""
+    """See if proj should be counted."""
     proj_ch_names = proj['data']['col_names']
     select = any(kk in ch_names for kk in proj_ch_names)
```

Comment:
return len(np.intersect1d(proj['data']['col_names'], ch_names)) > 0

seems more obvious to read for me